tests/conftest.py

```diff
@@ -1,14 +1,46 @@
+import contextlib
```

Comment:
The name of this file confuses me. Is it "config_test" or "confidence_test"? Or rather "mock_setup"?

Reply:
Yeah, I agree, unfortunately this is a somewhat required convention by pytest

Reply:
Aaaah, ok. Alright, then we keep it this way :)